modules/nw-sriov-configuring-device.adoc

```diff
@@ -78,6 +92,17 @@ For a Mellanox card to work in DPDK mode with Container-Native Virtualization (C
 If `isRDMA` flag is set to `true`, you can continue to use the RDMA enabled VF as a normal network device.
 A device can be used in either mode.
 ====
+endif::ocp-sriov[]
+ifdef::virt-sriov[]
+<13> The `vfio-pci` driver type is required for virtual functions in {VirtProductName}.
+<14> Optional. Specify whether to enable remote direct memory access (RDMA) mode. For a Mellanox card, set `isRdma` to `false`. The default value is `false`. 
++
+[NOTE]
```

Comment:
You could move this note out of the conditionals if you expect it to continue to be the same no matter how the rest of the annotation changes.

Reply:
Excellent point. I would have been far too superstitious to try that without the suggestion but it works as expected.